x-pack/plugins/observability/server/domain/services/composite_slo/validate_composite_slo.test.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  createWeightedAverageSource,
+  createCompositeSLO,
+} from '../../../services/composite_slo/fixtures/composite_slo';
+import { createSLO } from '../../../services/slo/fixtures/slo';
+import {
+  sevenDaysRolling,
+  thirtyDaysRolling,
+  weeklyCalendarAligned,
+} from '../../../services/slo/fixtures/time_window';
+import { validateCompositeSLO } from '.';
+
+describe('validateCompositeSLO', () => {
```

Comment:
What do you think about adding a successful case too?

Reply:
That can't hurt indeed :)